libs/eo/auth/data-access-security/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "projectType": "library",
+  "root": "libs/eo/auth/data-access-security",
+  "sourceRoot": "libs/eo/auth/data-access-security/src",
+  "prefix": "eo",
+  "targets": {
+    "test": {
+      "executor": "@nrwl/jest:jest",
+      "outputs": ["coverage/libs/eo/auth/data-access-security"],
+      "options": {
+        "jestConfig": "libs/eo/auth/data-access-security/jest.config.js",
+        "passWithNoTests": true
+      }
+    },
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "options": {
+        "lintFilePatterns": [
+          "libs/eo/auth/data-access-security/src/**/*.ts",
+          "libs/eo/auth/data-access-security/src/**/*.html"
+        ]
+      },
+      "outputs": ["{options.outputFile}"]
+    }
+  },
+  "tags": ["product:eo", "domain:auth", "type:data-access"]
```

Comment:
Is it intentional that you name the type tag `data-access` instead of `data-access-security`? 😄

Reply:
Yes, it's a `data-access` library (the library type) but it's specifically concerned with security. It's project name and path is then `<product>/<domain>/<type-prefix>-<library-name>`. We use library types to put static constraints enforced by lint rules on our inter-project dependencies as described in the readme file.